support-bundle.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: schemahero
+spec:
+  collectors: []
```

Comment:
IMO this should have collectors for the manager logs + statefulset status, at the very least
ideally the same for each namespace and the DB controller statefulsets